docs/devsite-help/rest-libraries.md

```diff
@@ -0,0 +1,168 @@
+
+OAuth authentication requires the end-user to interact with a web browser, which can only be done client-side.
+
+It is possible to use OAuth authentication for end users in a web application, but not using
+`GoogleWebAuthorizationBroker`. For these purposes, you can use one of the ASP.NET integration packages
+we provide, such as [`Google.Apis.Auth.AspNetCore3`](https://www.nuget.org/packages/Google.Apis.Auth.AspNetCore3).
+
+You can find general information about Google APIs and OAuth 2.0 in 
+[Using OAuth 2.0 to Access Google APIs](https://developers.google.com/identity/protocols/oauth2)
+and about integration with .NET client libraries in the [.NET guide for OAuth 2.0 and
+Google APIs](https://developers.google.com/api-client-library/dotnet/guide/aaa_oauth).
+
+## Specifying a retry policy
+
+By default, clients will only retry requests which result in an HTTP 503 response. These are retried with exponential backoff, starting with 250ms and doubling on each retry, for a maximum of 3 attempts.
+Retry policies can be configured on a per-client basis, as shown below:
+
+```csharp
+using Google.Apis.Drive.v3;
+using Google.Apis.Http;
+using Google.Apis.Util;
+using System.Net;
+
+BackOffHandler handler = new BackOffHandler(new BackOffHandler.Initializer(new ExponentialBackOff())
+{
+    HandleUnsuccessfulResponseFunc = response => response.StatusCode == HttpStatusCode.InternalServerError,
+    HandleExceptionFunc = exception => exception is HttpRequestException
+});
+
+DriveService service = new DriveService(new Google.Apis.Services.BaseClientService.Initializer
+{
+    // Prevent the default retry handling from also retrying failures.
+    DefaultExponentialBackOffPolicy = ExponentialBackOffPolicy.None,
+    ...
+});
+
+service.HttpClient.MessageHandler.NumTries = 5;
+service.HttpClient.MessageHandler.AddExceptionHandler(handler);
+service.HttpClient.MessageHandler.AddUnsuccessfulResponseHandler(handler);
+```
+
+## Integration with IHttpClientFactory
+
+For historical reasons, there are two (incompatible) `IHttpClientFactory` interfaces:
+
+- [`Google.Apis.Http.IHttpClientFactory`](https://github.com/googleapis/google-api-dotnet-client/blob/main/Src/Support/Google.Apis.Core/Http/IHttpClientFactory.cs) in the REST support library
+- [`System.Net.Http.IHttpClientFactory`](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.ihttpclientfactory) in .NET
+
+The .NET interface is widely used, and supported in the community with projects such as
+[Polly](https://github.com/App-vNext/Polly). Unfortunately, the REST support library predates this
+interface and it's can't be directly retro-fitted without breaking changes.
+
+The `HttpClientFromMessageHandlerFactory` provides a bridge between the two interfaces. Integration is still
```

Comment:
```suggestion
The `HttpClientFromMessageHandlerFactory` provides a bridge between the two interfaces. Usage is still
```

Reply:
I prefer integration here - if the issue is the duplication of "integration" later on the sentence, I think I'd probably prefer to remove it there rather than change this. Happy to chat about that or other options though.

Reply:
Yep, it's because of the duplication of integration, but I think it's actually fine, it really makes the most sense as it is.